bw2data/signals.py

```diff
@@ -1,5 +1,9 @@
 from blinker import signal
 
+database_saved = signal(
```

Comment:
`database_save` - is this master-level trolling? 😛

Reply:
There are two hard things in computer science… No but seriously, this name will change later.